besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -724,6 +724,12 @@ void setBannedNodeIds(final List<String> values) {
       description = "Stratum port binding (default: ${DEFAULT-VALUE})")
   private final Integer stratumPort = 8008;
 
+  @Option(
+      names = {"--miner-remote-sealers-limit"},
```

Comment:
Let's make this an `--X` option for the first few releases.  We can promote it later, but because this becomes part of the public API I want to have a window where it can be easily changed.

Reply:
Done